compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -353,16 +375,14 @@ struct QuantizeDequantizeWeights final : public luci::CircleNodeMutableVisitor<b
       auto input_node = node->arg(i);
       auto circle_node = loco::must_cast<luci::CircleNode *>(input_node);
 
-      // Check if this is already quantized
-      if (is_quantized(circle_node))
+      if (not is_quantizable(input_node))
         continue;
 
-      if (is_weights(circle_node))
-      {
-        auto circle_const = loco::must_cast<luci::CircleConst *>(circle_node);
+      if (not is_weights(circle_node))
```

Comment:
Maybe this check is not needed (duplicated check) because L357 also checks if node is weight.
(or maybe `is_quantizable` can be renamed to `is_quantizable_weight` to imply it also checks if the node is weight.)

Reply:
L357 does not check the node is weights. It only checks if the nodes is constant.

I've renamed the variable for clarity :)